src/conf.c

```diff
@@ -57,6 +57,19 @@
 
 int parse_interfaces( conf_t *conf, char *s );
 
+#define axel_fscanf( _fp, ... )				\
```

Comment:
This could be made a function by using `vfscanf`.

Reply:
I did not make it a function, because this macro will also call return for conf_loadfile(). In alternative I can make it a function and return in conf_loadfile() when this new function returns -1, if you prefer.

Reply:
Yes, please.